app/views/user_mailer/welcome_email_driver.html.erb

```diff
@@ -21,12 +21,4 @@
 <% end %>
```

Comment:
I don't see anything wrong with the changes to this file, but noticed that up on L12 it mentions "November 8" in particular. Looks like there are a few places in the code base with that hard-coded 2016 date.